src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ApplicationService.kt

```diff
@@ -1047,10 +1047,11 @@ class ApplicationService(
   fun getOfflineApplicationsForCrn(crn: String, serviceName: ServiceName) =
     offlineApplicationRepository.findAllByServiceAndCrn(serviceName.value, crn)
 
-  fun getApplicationTimeline(applicationId: UUID): List<TimelineEvent> {
+  fun getApplicationTimeline(applicationId: UUID, user: UserEntity): List<TimelineEvent> {
     val domainEvents = domainEventService.getAllDomainEventsForApplication(applicationId)
+    val canManageAppeals = user.hasRole(UserRole.CAS1_APPEALS_MANAGER)
```

Comment:
I think it'd be good going forward if we could have one source of truth for what a given user can do (even if it is just based on one role)

Reply:
hmm, I'm not aware of this requirement to limit access to the appeal page. Where does that come from? What's the information on that view which shouldn't be seen by applicants?

Reply:
Yeah, it's this line https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/e8c8d3c9bf64c70cbf456d538b4b4487180ec5e2/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AppealService.kt#L40

Reply:
It does seem a bit weird though - I'll remove the transformer requirement in this PR, and open another PR to remove the access limitation

Reply:
PR here: https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1510